### PR TITLE
Update core.lua for expression2 to add E:ops()

### PR DESCRIPTION
**Pull request by sammyt291**
Add E:ops()
ops() exists currently but no E:ops()

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/core.lua [hunk 1]
```diff
@@ -389,6 +389,11 @@ e2function number ops()
 	return round(self.prfbench)
 end
 
+e2function number entity:ops()
+	if not IsValid(this) or this:GetClass() ~= "gmod_wire_expression2" or not this.context then return 0 end
+	return this.context.prfbench
```
Comment (Divran) [inline lua/entities/gmod_wire_expression2/core/core.lua]: should it not use `round(this.context.prfbench)` like the function above?

Comment (sammyt291) [inline lua/entities/gmod_wire_expression2/core/core.lua]: Yes it most likely should, oversighted that, thank you.

File: lua/entities/gmod_wire_expression2/core/core.lua [hunk 2]
```diff
@@ -525,4 +530,4 @@ registerOperator("include", "", "", function(self, args)
 		self:PopScope()
 		self:LoadScopes(OldScopes)
 	end
-end)
\ No newline at end of file
+end)
```
Comment (TomyLobo) [inline lua/entities/gmod_wire_expression2/core/core.lua]: Can you double-check that there is an empty line at the end of the file?

Comment (Divran) [inline lua/entities/gmod_wire_expression2/core/core.lua]: according to this diff, there is, but why does it matter?

Comment (oldmud0) [inline lua/entities/gmod_wire_expression2/core/core.lua]: ^

Comment (sammyt291) [inline lua/entities/gmod_wire_expression2/core/core.lua]: There is indeed an empty line on ln #534

Comment (Divran) [inline lua/entities/gmod_wire_expression2/core/core.lua]: @TomyLobo ok then, good to merge?

Comment (TomyLobo) [inline lua/entities/gmod_wire_expression2/core/core.lua]: sure

**Pull request merged**